Client/src/app/components/historyComponent/history.component.ts

```diff
@@ -0,0 +1,34 @@
+import {Component, EventEmitter, Output} from '@angular/core';
+import {HttpService} from '../http.service';
+import {PlayerService} from '../player.service';
+
+@Component({
+  selector: 'history-component',
+  templateUrl: './history.component.html',
+  styleUrls: ['./history.component.less']
+})
+
+export class HistoryComponent {
+  historyReady = false;
+
+  constructor(private httpService: HttpService,
+              private playerService: PlayerService) { }
+
+  async ngOnInit() {
+    await this.getAllHistory();
+  }
+
+  async getAllHistory() {
+    await this.httpService.getHistory();
```

Comment:
I don't like that the 'history' list is a member of httpService instead of the HistoryComponent. In my opinion, getHistory() should return the history list and this component should store it. It will make this code a little bit more clear, make httpService stateless (which will allow it to be used in another components in the future for instance) and will remove the bindings from the html to httpService which is a bit weird.